### DISCUSSION TITLE
Error handling

### DISCUSSION THREAD

**Discussion by simemese** [Azure/azure-functions-sql-extension#242] [Q&A]
Hi, how can you add custom error handling when using input binding? Let's say one of the columns doesn't exist or the type is wrong.

**Comment by Charles-Gagnon** [comment]
Could you explain your situation a bit more to try and help us understand what this would look like? The scenarios you listed would generally be due to an error in the function code itself or the parameters being passed to the binding - what kind of "error handling" are you thinking you would like to see?
